core/src/main/scala/scala/pickling/generator/Macros.scala

```diff
@@ -86,6 +88,25 @@ private[pickling] trait PicklingMacros extends Macro with SourceGenerator with T
         tree
     }
   }
+
+  def debugPicklerUnpickler[T: c.WeakTypeTag]: c.Tree = preferringAlternativeImplicits {
+    val tpe = computeType[T]
+    checkClassType(tpe)
+    val sym = symbols.newClass(tpe)
+    val impl = PicklingAlgorithm.run(generator)(sym, logger)
+    //System.err.println(impl)
+    val tree2 = impl map generatePicklerUnpicklerClass[T]
+    tree2 match {
+      case None =>
+        c.error(c.enclosingPosition, s"Failed to generate pickler/unpickler for $tpe")
+        ???
+      case Some(tree) =>
+        //System.err.println(s" --=== $tpe ===--\n$tree\n --=== / $tpe ===--")
+        val treeString = s"$tree"
+         q"$treeString"
```

Comment:
quasiquotes would embedd the tree in that case.  This is a fallout form magic quasi-quote treatment of strings -> string literals. I could do `${tree.toString}` perhaps....